docs/src/increase-observability/logs/forward-logs.md

```diff
@@ -24,6 +24,14 @@ To enable log forwarding in a project, you need to be a [project admin](../../ad
 You also need your project to have the capability for log forwarding.
 To get that, contact [support](https://console.platform.sh/-/users/~/tickets/open).
 
+{{< note >}}
+
+The log forwarding feature currently does not preserve the original log level. However, to preserve the original log level, there are two alternatives:
```

Comment:
`By default, the log forwarding feature will log everything at the INFO level, disregarding the original level used. In order to preserve the original log level you can use the language specific syslog module/package for logging.`

Maybe add some examples of how you'd do this for PHP and Python?